blogposts/2021/workspaces-of-sourcegraph.md

```diff
@@ -0,0 +1,225 @@
+- Plugins/features: I'll just install as needed, but mostly LSPs, :emojisuggest:, TODO Highlight, colorize, .editorconfig., Fira Code but have been trying, Cascadia Code. Shoutout for Inconsolata before I moved to fonts with ligatures.
+- Console: zsh + Starship Theme + fzf + bat = :heart:
+
+![Workspace of Gonzalo Peci, Engineering Manager](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/gonza.png)
+
+
+## Thorsten Ball, Software Engineer
+
+- Environment: tmux in a fullscreen Kitty window that covers everything
+- Editor: Neovim (sometimes VS Code with Vim keybindings for TypeScript/Rust)
+- Mode: 90% of the time it’s light with the Lucius colorscheme, 10% it’s dark with the VS Code dark colorscheme
+- Font: JetBrains Mono
+- Shell: zsh
+- Vim configuration: https://github.com/mrnugget/vimconfig
+- Dotfiles: https://github.com/mrnugget/dotfiles
+
+I basically do everything either in tmux or in the browser. In tmux I spawn lots of windows/shells as I need them, but have different sessions for work, private recreational programming, and other things.  In the browser I always have 3 pinned tabs (for gmail accounts and other email account). I also use Alfred a lot (to search Sourcegraph for example).
+
+![Workspace of Thorsten Ball, Software Engineer](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/thorsten.png)
+
+
+## TJ Kandala, Software Engineer
+
+- Editor: VS Code
+- Mode: Dark
+- Theme: SynthWave 84
+- Extensions: Bracket Pair Colorizer, GitLens, Sourcegraph
+- Misc: Fira Code font, Tilix terminal emulator
+
+![Workspace of TJ Kandala, Software Engineer](https://storage.googleapis.com/sourcegraph-assets/blog/workspaces-of-sourcegraph/tj.png)
+
+
+## Quinn Keast, Product Designer
+
+- Nova (Cobalt theme) and VSCode (Cobalt2)
+- Typeface: Operator Mono
+- Sourcegraph light theme
+- Desktop organization: Browser and reference materials on the left, Figma or full-screen editors on the main display. Sometimes I use terminals in the IDE, and other times I use iTerm2.
+
+I'm experimenting with the Nova editor alongside VSCode, which replaced Sublime Text 2. I don't have any standard window setup, shifting as I need for any given design, research, or programming need. I've been using the fantastic Operator Mono font with ligatures and italics for years.
+
+![Primary screen workspace organization of Quinn Keast, Product Designer](https://sourcegraphstatic.com/blog/workspaces-of-sourcegraph/Quinn-Keast-1.png)
+![Secondary screen workspace organization of Quinn Keast, Product Designer](https://sourcegraphstatic.com/blog/workspaces-of-sourcegraph/Quinn-Keast-2.png)
+
+# What does your editor look like? 
+
+What editors, themes and fonts do you use? DM or tweet us at [@srcgraph](https://twitter.com/srcgraph) to share your thoughts and images. /Make sure to not include sensitive information in any screens you share!/
```

Comment:
```suggestion
We want to see your setup too! Tweet it to us [@srcgraph](https://twitter.com/srcgraph).
```

Reply:
@content-hacker Does it make sense to repeat the call for submissions? If so I changed the copy to not duplicate the one above.